api/v1beta1/manila_webhook.go

```diff
@@ -82,6 +94,11 @@ func (spec *ManilaSpec) Default() {
 			spec.ManilaShares[key] = manilaShare
 		}
 	}
+
+	if (spec.DBPurge.Age == 0) && (spec.DBPurge.Schedule == "") {
```

Comment:
See Gorka's comment for the corresponding code in https://github.com/openstack-k8s-operators/cinder-operator/pull/295

Reply:
+1 and thanks for sharing, just commented and updated the patch. I think it's ok to process these two struct fields independently.